python/simpler/worker.py

```
@@ -1690,8 +1725,113 @@ def handle_control(sub_cmd: int) -> tuple[int, str]:  # noqa: PLR0912 -- one bra
+                control_state = _mailbox_load_i32(state_addr)
+                with action_cv:
+                    if control_state == _SHUTDOWN:
+                        actions.append(("shutdown", -1, None))
+                        action_cv.notify()
+                        return
+                    if control_state == _CONTROL_REQUEST and not control_queued:
+                        control_queued = True
+                        actions.append(("control", -1, None))
+                        action_cv.notify()
+
+                for index, (frame_buf, frame_addr) in enumerate(zip(frame_bufs, frame_addrs)):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)